packages/winpanda/extra/src/winpanda/tests/unit/test_cfgm_cfgm.py

```diff
@@ -0,0 +1,17 @@
+import mock
+import unittest
+
+from cfgm.cfgm import PkgConfManager
+
+
+class TestPkgConfManager(unittest.TestCase):
+    """DC/OS package configuration files manager unit tests."""
+
+    @mock.patch('cfgm.cfgm.isinstance', return_value=True)
+    def test_config_manager_init(self, *args):
+        """
+        Check PkgConfManager __str__ magic method return value.
+        """
```

Comment:
Could you make it more stylish, please?
Ref: https://www.python.org/dev/peps/pep-0257/#one-line-docstrings

Reply:
done